src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Lifted/ObjectLifetimeTests.Lifted/ObjectLifetimeTests.Lifted.csproj

```diff
@@ -0,0 +1,116 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+        <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+        <RootNamespace>ObjectLifetimeTests.Lifted</RootNamespace>
+        <ApplicationManifest>app.manifest</ApplicationManifest>
+        <Platforms>x86;x64;arm64</Platforms>
+        <RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+        <EnablePreviewMsixTooling>True</EnablePreviewMsixTooling>
+        <PublishProfile>win10-$(Platform).pubxml</PublishProfile>
+        <PrepareMsixPackageDependsOn></PrepareMsixPackageDependsOn>
+        <PublishReadyToRun>false</PublishReadyToRun>
+        <ProjectReunionVerifyWinrtRuntimeVersion>false</ProjectReunionVerifyWinrtRuntimeVersion>
+    </PropertyGroup>
+    <PropertyGroup>
+        <DefineConstants>LIFTED</DefineConstants>
```

Comment:
I doubt we'll ever test system in C#/WinRT - can we remove the LIFTED define and all conditional code to simplify?